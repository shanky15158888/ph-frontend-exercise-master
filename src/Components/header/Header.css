header {
    border-bottom: 2px solid #dfdfe1;;
    height: 52px;
  }
  .logo .partnerText {
    font-weight: 500;
    display: inline-block;
    color: #959599;
    position: relative;
    top: 5px;  
  }
  .logo .logo-section {
    display: inline-block;
    height: 45px;
    margin: 10px 5px auto 50px
  }
  .account {
    margin: 10px 100px auto auto;
    float: right;
  }
  .nav-StoreLogo-svg {
    display: inline-block;
    vertical-align: middle;
  }
  .accountIcon {
    height: 24px;
    width: 24px;
    display: inline-block;
  }
  #navbarSupportedContent {
    display: inline-block;
  }
  .nav-link, .dropdown-item {
    color: #211e22;
    font-size: 14px;
  }
  