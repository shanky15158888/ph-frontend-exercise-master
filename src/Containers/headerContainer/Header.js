import React from 'react'
import { connect } from 'react-redux'
import Header from '../../Components/header/Header'

const mapStateToProps = (state, ownProps) => {
    return {
        width: state.MenubarReducer.width
    }
}

export default connect(mapStateToProps)(Header)
