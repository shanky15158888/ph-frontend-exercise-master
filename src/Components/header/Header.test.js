import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';

test('Header testing', () => {
  const component = renderer.create(
    <Header width="18"/>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});