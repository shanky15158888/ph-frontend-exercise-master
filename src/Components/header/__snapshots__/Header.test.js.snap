// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header testing 1`] = `
<header
  style={
    Object {
      "width": "82%",
    }
  }
>
  <nav
    className="logo"
  >
    <div
      className="logo-section"
    >
      <svg
        aria-label="Wayfair"
        className="nav-StoreLogo-svg"
        height="32"
        role="img"
        style={
          Object {
            "display": "inline-block",
            "verticalAlign": "middle",
          }
        }
        viewBox="0 0 1299.3 285"
        xmlns="http://www.w3.org/2000/svg"
      >
        <title>
          Wayfair
        </title>
        <path
          d="M456 69.2h-57.2c-5.2 0-9.4 4.2-9.4 9.4v12.8c.1 3.4 2.9 6.1 6.3 6.1h9.4c.3 0 .5.1.7.4.2.3.3.7.2 1L383.6 166c-.2.4-.6.7-1.1.6-.4 0-.8-.3-.9-.6l-33.5-90.9V75c-.9-2.2-3-3.8-5.4-3.9h-20.2c-2.4.2-4.5 1.7-5.5 3.9v.1L283.5 166c-.1.4-.5.7-.9.7-.5 0-.9-.2-1.1-.6l-22.4-67.2c-.1-.3 0-.7.2-1 .2-.2.4-.4.7-.4h9.4c3.4 0 6.2-2.7 6.3-6.1V78.6c0-5.2-4.2-9.4-9.4-9.4h-57.2c-2 0-3.9 1-5.1 2.6-1.1 1.5-1.5 3.5-.9 5.3.8 2.3 1.8 5.1 2.7 8s1.9 5.6 2.8 8.1c.8 2.5 3.1 4.2 5.8 4.3h9.6c.5 0 1 .3 1.1.8L269.6 229c.8 2.5 3.1 4.3 5.8 4.3h13.8c2.6 0 4.9-1.6 5.8-4 12.2-35.4 35.6-102.8 36.5-105.1.1-.5.6-.9 1.1-.9.5 0 1 .3 1.1.8.9 2.4 24.3 69.8 36.5 105.1.9 2.4 3.2 4 5.8 4.1h13.8c2.7 0 5-1.8 5.8-4.3L440 98.3c.4-.8.9-.8 1.1-.8h9.6c2.6 0 4.9-1.7 5.8-4.2.9-2.5 1.8-5.4 2.8-8.1s2-5.7 2.8-8c.6-1.8.2-3.8-.9-5.3-1.3-1.7-3.1-2.7-5.2-2.7zM1125.6 206.9h-10.4c-1.6 0-2.9-1.3-2.9-3V75.1c0-2.3-1.9-4.2-4.3-4.1H1077.6c-1.3 0-2.5.6-3.3 1.6l-12.6 15.7c-1.5 1.8-1.1 4.5.7 6 .7.6 1.6.9 2.4.9h3.9c1.6.1 2.9 1.4 3 3v105.5c0 .8-.3 1.7-.8 2.4-3.3 4.3-10.6 13.4-14.2 17.9-1.6 1.8-1.4 4.5.4 6 .8.7 1.7 1 2.7 1.1h53c1.1.1 2.2-.3 3-1.2l12.4-15.5c2.9-3.4.9-7.5-2.6-7.5z"
          fill="#7f187f"
        />
        <circle
          cx="1091.8"
          cy="30.8"
          fill="#7f187f"
          r="22.8"
        />
        <path
          d="M1292.4 84.3c0-2.2-1.2-4.3-4.1-4.3h-5.3v13.9h2v-5.1h1.8l3.5 5.1h2.3l-3.7-5.3c2.6-.2 3.5-2.3 3.5-4.3zm-7.4 2.5v-5.1h3.5c1.3.1 2.2 1.1 2.2 2.4v.2c.1 1.3-.8 2.4-2 2.5H1285z"
          fill="#7f187f"
        />
        <path
          d="M1287.3 74.9c-6.6 0-12 5.4-12 12s5.4 12 12 12 12-5.4 12-12-5.3-12-12-12zm0 22.2c-5.6 0-10.2-4.5-10.2-10.1v-.1c-.1-5.5 4.3-10.1 9.8-10.2h.4c5.6 0 10.2 4.6 10.2 10.2s-4.5 10.2-10.2 10.2zM605.4 207.5c-8.8.4-13.2-3.7-13.2-13.8v-72.3c0-31.6-26.5-51.7-56.4-51.7-46.4 0-63.1 22-65 46.6-.1 2.4 1.7 4.4 4 4.5h23.1c2.2-.1 4-1.8 4.1-3.9 1.2-12.8 9.8-22.2 24.2-22.2 15.9 0 25.5 8.1 25.5 26.3v15.7h-8.3c-28.5 0-48.1 3.7-61.3 12-13.2 8.3-19.8 23.6-19.8 41.5 0 13.2 4.5 24 13.6 32.6s19.7 12.4 33.6 12.4c8.3 0 16.5-1.9 24-5.5 7.4-3.3 14.2-7.7 20.2-13 1.2-1.1 3.1-1.1 4.2.1.2.2.4.4.5.7 6.5 11.4 13.6 15.1 27.1 15.1 12.2 0 18.1-11.4 23.4-18.3 2-2.7-.2-7-3.5-6.8zm-53.5-11.6h-.2c0 1-1.6 2.4-2.2 3-3.7 3.1-7.8 5.8-12.2 7.9-4 1.7-8.3 2.5-12.6 2.4-6.3 0-12.6-2.4-16.7-6.9-4.2-4.6-6.4-10.7-6.3-16.9 0-9.4 3.5-15.7 10.6-20.4s18.7-6.9 32.6-6.9h6.9l.1 37.8zM1040.1 207.5c-8.8.4-13.2-3.7-13.2-13.8v-72.3c0-31.6-26.5-51.7-56.4-51.7-46.4 0-63.1 22-65 46.6-.1 2.4 1.7 4.4 4 4.5h23.1c2.2-.1 4-1.8 4.1-3.9 1.2-12.8 9.8-22.2 24.2-22.2 15.9 0 25.5 8.1 25.5 26.3v15.7h-8.2c-28.5 0-48.2 3.7-61.3 12-13.2 8.3-19.9 23.6-19.9 41.5 0 13.2 4.5 24 13.6 32.6s19.7 12.4 33.6 12.4c8.3 0 16.5-1.9 24-5.5 7.4-3.3 14.2-7.7 20.2-13 1.2-1.1 3.1-1.1 4.2.1.2.2.4.4.5.7 6.5 11.4 13.6 15.1 27.1 15.1 12.2 0 18.1-11.4 23.4-18.3 2-2.7-.2-7-3.5-6.8zm-53.4-11.6h-.2c0 1-1.6 2.4-2.2 3-3.7 3.1-7.8 5.8-12.2 7.9-4 1.7-8.3 2.5-12.6 2.4-6.3 0-12.6-2.4-16.7-6.9-4.2-4.6-6.4-10.7-6.3-16.9 0-9.4 3.5-15.7 10.6-20.4s18.7-6.9 32.6-6.9h6.9v37.8zM1255.6 70.4c-2.1-1.1-7.4-1.6-12-1.6-7.4 0-13.3 1-18.4 3.2l-.4.2c-1.9.9-3.8 1.9-5.5 3.2-6.5 4-12.2 9-17.2 14.8 0 0-2.2 1.7-2.6-1.1V74.6c0-2.3-1.9-4.2-4.2-4.2h-30.4c-1.2 0-2.4.5-3.1 1.5 0 0-9.6 11.9-12.7 16-1.9 2.4-.7 6.9 3.3 6.9h3.8c1.6 0 3 1.4 3 3v106c0 .8-.2 1.7-.7 2.4-3 3.7-10.6 13.3-14.3 18-1.4 1.9-1.1 4.5.8 6 .7.6 1.6.9 2.5.9h65.7c2.3 0 4.2-1.9 4.2-4.2V211c0-2.3-1.9-4.2-4.2-4.2h-10.4c-1.7 0-3-1.3-3-3v-61.7c1.2-29.3 18.7-38.3 27.1-40.9l1.2-.4.3-.1c2.3-.6 4.6-.9 7-.9 10 0 13.7 1.5 19.2 2.9 2.3.6 4.2-1.9 4.2-4.2v-24c-.2-1.9-1.7-3.4-3.2-4.1zM898.3 70.9h-47.6c-1.6 0-3-1.4-3-3V44.4c0-11.4 5.9-22.8 18.1-22.8 13.8 0 15.9 11.6 16.5 20.4.1 2.2 1.9 4 4.1 4.1h19.7c2.2-.1 4-1.9 3.9-4.2-.4-29.4-19.1-42-49.1-42-18.3 0-30.3 5.7-39.3 16.7S808 39.9 808 61.9v5.9c0 1.6-1.3 2.9-2.9 2.9l-81.3.2c-2.3-.1-4.3 1.7-4.3 4.1v16c0 2.4 2 4.3 4.3 4.3h8.7c1.7 0 3 1.4 3 3 0 .4-.1.8-.2 1.1 0 0-22.4 52.1-31.4 72.9-.7 1.5-2.4 2.2-3.9 1.6-.7-.3-1.2-.9-1.6-1.6-9.2-21-31.8-72.9-31.8-72.9-.6-1.5.1-3.3 1.6-3.9.3-.1.7-.2 1.1-.2h6.1c2.4 0 4.3-1.9 4.3-4.3V75.1c0-2.3-1.9-4.2-4.3-4.1h-73.6c-2.4 0-4.3 2-4.2 4.4 0 .7.2 1.3.5 1.9 2.2 3.9 7.1 13.2 8.5 15.9.7 1.3 2.1 2.1 3.5 2.2h14.2c1.2 0 2.3.7 2.8 1.8 0 0 52.3 117.4 54.2 121.5 2 5.2 2 11-.1 16.3-2.4 9.6-18.5 26.8-27.4 23.7-6.1-2.1-6.9-13.2-14.2-14.6-1.4-.2-2.7-.1-4 .5-1.3.5-13.2 15.8-15.8 18.5-1.3 1.7-1.3 4 0 5.7 6.5 6.7 17.4 15.7 33.2 16.3 19.4.8 38.5-1.8 56.2-35.4 0 0 56.3-137.6 63-152.4.6-1 1.6-1.6 2.8-1.6h29.7c1.6 0 3 1.3 3 2.9v105.7c0 .9-.3 1.7-.8 2.4-3.5 4.5-10.6 13.4-14.2 17.9-1.5 1.8-1.2 4.5.6 5.9.8.6 1.7 1 2.7.9h66.4c2.4 0 4.3-1.9 4.3-4.3v-15.7c0-2.4-1.9-4.3-4.3-4.3H851c-1.6 0-2.9-1.3-2.9-3V98.2c0-1.6 1.3-2.9 2.9-2.9h35.1c1.3 0 2.5-.6 3.3-1.6 3-3.9 9-11.8 12.2-15.9 1.4-1.9 1.1-4.5-.8-6-.7-.5-1.6-.9-2.5-.9z"
          fill="#7f187f"
        />
        <path
          d="M181.6 121.8l-11.5-43.2c-.4-1.5-1.5-2.6-3-3l-43.2-11.5c-1.4-.4-3 0-4 1.1l-29 29 60.7 60.7 29-29c1-1.1 1.4-2.7 1-4.1z"
          fill="#82d170"
        />
        <path
          d="M.1 187.9l11.5 43.2c.4 1.5 1.5 2.6 3 3l43.3 11.5c1.4.4 3 0 4-1.1l29-29-60.6-60.7-29 29c-1.1 1.1-1.5 2.6-1.2 4.1z"
          fill="#f8cf1c"
        />
        <path
          d="M57.8 64.1L14.6 75.6c-1.5.4-2.6 1.5-3 3L.1 121.8c-.4 1.4 0 3 1.1 4l29 29 60.6-60.6-29-29c-1-1.1-2.5-1.5-4-1.1z"
          fill="#c6d636"
        />
        <path
          d="M180.1 183.4l-28.6-28.6-60.6 60.6 28.6 28.6c1.3 1.3 3.2 1.8 5 1.4l42.1-11.2c1.8-.5 3.2-1.9 3.7-3.7l11.2-42.1c.4-1.8-.1-3.7-1.4-5z"
          fill="#d57dea"
        />
        <path
          d="M90.9 94.2l-60.6 60.6 60.6 60.6 60.7-60.6-60.7-60.6z"
          fill="#7f187f"
        />
        <path
          d="M181.6 121.8l-11.5-43.2c-.4-1.5-1.5-2.6-3-3l-43.2-11.5c-1.4-.4-3 0-4 1.1l-29 29 60.7 60.7 29-29c1-1.1 1.4-2.7 1-4.1z"
          fill="#82d170"
        />
        <path
          d="M.1 187.9l11.5 43.2c.4 1.5 1.5 2.6 3 3l43.3 11.5c1.4.4 3 0 4-1.1l29-29-60.6-60.7-29 29c-1.1 1.1-1.5 2.6-1.2 4.1z"
          fill="#f8cf1c"
        />
        <path
          d="M57.8 64.1L14.6 75.6c-1.5.4-2.6 1.5-3 3L.1 121.8c-.4 1.4 0 3 1.1 4l29 29 60.6-60.6-29-29c-1-1.1-2.5-1.5-4-1.1z"
          fill="#c6d636"
        />
        <path
          d="M180.1 183.4l-28.6-28.6-60.6 60.6 28.6 28.6c1.3 1.3 3.2 1.8 5 1.4l42.1-11.2c1.8-.5 3.2-1.9 3.7-3.7l11.2-42.1c.4-1.8-.1-3.7-1.4-5z"
          fill="#d57dea"
        />
        <path
          d="M90.9 94.2l-60.6 60.6 60.6 60.6 60.7-60.6-60.7-60.6z"
          fill="#7f187f"
        />
      </svg>
    </div>
    <div
      className="partnerText"
    >
      PARTNER HOME
    </div>
    <div
      className="account"
    >
      <div
        className="navbar-collapse"
        id="navbarSupportedContent"
      >
        <div
          className="navbar-nav mr-auto"
        >
          <div
            className="nav-item dropdown"
          >
            <a
              aria-expanded="false"
              aria-haspopup="true"
              className="nav-link dropdown-toggle"
              data-toggle="dropdown"
              href="#"
              id="navbarDropdownMenuLink"
            >
              Harpreet
            </a>
            <div
              aria-labelledby="navbarDropdownMenuLink"
              className="dropdown-menu"
            >
              <a
                className="dropdown-item"
                href="#"
              >
                Account Settings
              </a>
              <a
                className="dropdown-item"
                href="#"
              >
                User Management
              </a>
              <a
                className="dropdown-item"
                href="#"
              >
                My Team
              </a>
              <a
                className="dropdown-item"
                href="#"
              >
                Logout
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        className="accountIcon"
      >
        <svg
          aria-label="My Account"
          className="pl-BaseIcon BaseIcon pl-BaseIcon--scalable MyAccountLinkCard-icon"
          data-hb-id="pl-icon"
          focusable="false"
          role="img"
          viewBox="2 2 24 24"
        >
          <title>
            My Account
          </title>
          <path
            d="M14 4.5a9.5 9.5 0 109.5 9.5A9.51 9.51 0 0014 4.5zM9.26 21.05v-2.17a3.37 3.37 0 013.36-3.36h2.74a3.37 3.37 0 013.36 3.36v2.19a8.47 8.47 0 01-9.48 0zM14 14.5a2.5 2.5 0 112.5-2.5 2.5 2.5 0 01-2.5 2.5zm5.73 5.76v-1.38a4.37 4.37 0 00-3.44-4.26A3.45 3.45 0 0017.5 12a3.5 3.5 0 00-7 0 3.45 3.45 0 001.21 2.62 4.37 4.37 0 00-3.44 4.26v1.38a8.5 8.5 0 1111.46 0z"
          />
        </svg>
      </div>
    </div>
  </nav>
</header>
`;
